app/views/index.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Recent Peeps!</h1>
+
+<ul id = 'peeps'>
+  <% @peeps.reverse_each do |peep| %>
+  <li>
+    Time: <%= peep.peep_time %>
+    <br>
```

Comment:
generally avoid `<br>` tags as they imply layout, which is not the job of HTML.  HTML should only be concerned with semantic structure.